schema/crdb/dbinit.sql

```diff
@@ -5693,6 +5693,71 @@ CREATE UNIQUE INDEX IF NOT EXISTS one_record_per_volume_resource_usage on omicro
     region_snapshot_snapshot_id
 );
 
+CREATE TABLE IF NOT EXISTS omicron.public.audit_log (
+    id UUID PRIMARY KEY,
+    time_initialized TIMESTAMPTZ NOT NULL,
```

Comment:
Thinking of renaming this to `time_started` because it describes the user-facing meaning of it. "Initialization" is the operation of creating an entry in this table. I could theoretically only change the name in the API view struct, but I tend to find things easier to follow if we use the same name all the way down.